src/commands/code-analyzer/rules.ts

```diff
@@ -55,6 +62,9 @@ export default class RulesCommand extends SfCommand<void> implements Displayable
 
 	public async run(): Promise<void> {
 		const parsedFlags = (await this.parse(RulesCommand)).flags;
+		if (parsedFlags.target && !parsedFlags.workspace) {
+			parsedFlags.workspace = ['.'];
```

Comment:
In `rules`, `--workspace` and `--target` are both optional, raising the possibility that `--target` could be provided but not `--workspace`.
Functionally, it doesn't really matter what we do in that case, but I made the decision to have the workspace default to the current directory.